src/identity/IdentityPage.test.tsx

```diff
@@ -32,9 +32,12 @@ const createSharingCodeForUserIdMock = createSharingCodeForUserId as jest.Mocked
   typeof createSharingCodeForUserId
 >;
 
+const RealDate = Date;
```

Comment:
Why not use a library for this? 
i.e: https://github.com/cov-clear/backend/blob/d410b6a417924bdd19f93f8e5927fecffe40c5e9/src/domain/model/accessPass/AccessPass.test.ts#L2

Reply:
Ah cool spot - i found it a bit handy to have access to the real date as well to do relative time, but can do without.

Reply:
Thanks much cleaner now